Gemfile

```diff
@@ -69,7 +69,9 @@ gems = {}
 gems['facter'] = location_for(facter_version) if facter_version
 gems['hiera'] = location_for(hiera_version) if hiera_version
 gems['puppet'] = location_for(puppet_version) if puppet_version
-gem 'bolt', '~> 0.23.0'
+# gem 'bolt', '~> 1.0.0', require: false
+gem 'bolt', git: "https://github.com/donoghuc/bolt.git", ref: 'BOLT-915'
```

Comment:
This could be updated to track puppetlabs/bolt#master if we wanted to get it in. Do we need beaker-task_helper?

Reply:
Oh, we use it for inventory.

Reply:
I'm fine with it tracking master. Updated.